src/macro.ts

```diff
@@ -129,14 +130,20 @@ export default class Macro extends StrictMacro {
   hardKill(): this {
     if (myClass() === $class`Grey Goo`) return this;
 
+    const [bestKillSpell, otherKillSpell] =
+      myClass() === $class`Pastamancer`
+        ? $skills`Weapon of the Pastalord, Saucegeyser`
```

Comment:
Minor but something inside me tells me not to use `$skills` for constructing a tuple. I can't quite explain why, something around it being less clear that its size is bounded.

Reply:
I'll merge anyway, but still